### ISSUE TITLE
LC0024 semicolon is needed after procedure declaration in controladdin

### ISSUE THREAD

**Issue by finnpedersenkazes** [StefanMaron/BusinessCentral.LinterCop#1051]
Hi, 

The LC0024 warning should not be given in a controladdin object. 

Procedure declarations in the controladdin are without a body. 

Like this 

```
controladdin TimerControl
{
    ....

    internal procedure StartTimer(milliSeconds: Integer); <--- semicolon is needed
    internal procedure StopTimer(); <--- and here 
    event ControlAddInReady();
    event RefreshPage();
}
```

**Comment by Arthurvdv** [comment]
Thank you for raising this issue and you're right this is indeed a false positive.

> A semicolon (statement separator) suggests that a new statement is coming up.

The [LC0042](https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0024) documentation indicates that the semicolon usage is valid in this context.

It appears this extends not only to `controladdin` objects but also to `interface` objects. So I'm thinking of adapting the rule to exclude procedure declarations without a body.

**Comment by Arthurvdv** [comment]
This was easier than I expected and should now be resolved in the Pre-release [v0.38.1](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.38.1) version of the LinterCop.

**Comment by Arthurvdv** [comment]
The version [v0.39.0](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.39.0) of LinterCop has been released as the latest, where this issue is now resolved.
